Networking/Extensions/DecodedExtension.swift

```diff
@@ -29,6 +29,17 @@ public extension Decoded {
     
 }
 
+public extension Decodable where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
right, I guess we couldo do String, Numbers (Int, Double, Float, UInt) and Bool.